AutoPartsShop.WebUI/Services/Interfaces/IProductService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using AutoPartsShop.Models.DTOs;
+
+namespace AutoPartsShop.WebUI.Services.Interfaces
+{
+    public interface IProductService
+    {
+        Task<IEnumerable<ProductDTO>> GetItems();
```

Comment:
Same comment about naming applies to these services as well. The methods name should reflect the entity we are using. So `GetItems` -> `GetProducts` and so on.